Introduce self-monitor dockerfile (#421)
kyma-project/third-party-images: telemetry-self-monitor/README.md

# Telemetry Self Monitor Docker Image

> The container image is built based on latest [prometheus lts image](https://prometheus.io/docs/introduction/release-cycle/).

Reviewer: ```suggestion
The container image is built based on the latest [Prometheus LTS image](https://prometheus.io/docs/introduction/release-cycle/).
```

> Additionally, there is [plugins.yml](./plugins.yml) which contains the list of the plugins required. 

Reviewer: ```suggestion
Additionally, there is a [plugins.yml](./plugins.yml) file, which contains the list of the required plugins. 
```

## Build locally

> To build the image locally, execute the following command, entering the proper versions taken from the `envs` file:

Reviewer: ```suggestion
To build the image locally with the versions taken from the `envs` file, run:
```
